test/apps/gups.c

```diff
@@ -264,7 +264,7 @@ UpdateTable(uint64_t *Table,
             int64_t niterate,
             int use_lock)
 {
-  uint64_t iterate;
+  int64_t iterate;
```

Comment:
Hmm...I think you're right.  I made this change to quiet a warning on line 278 because the conditional was comparing signed with unsigned, but the indentation suggests that LocalTableSize used to be a uint64_t and was changed to signed.  Changing it back seems like a much better fix for the sign issue.